src/lib/krb5/os/accessor.c

```diff
@@ -41,7 +41,7 @@ krb5_error_code KRB5_CALLCONV
 krb5int_accessor(krb5int_access *internals, krb5_int32 version)
 {
     if (version == KRB5INT_ACCESS_VERSION) {
-#if DESIGNATED_INITIALIZERS
+#ifdef DESIGNATED_INITIALIZERS
```

Comment:
Oops, I've done the wrong thing.  I'll fix it in a moment, but for posterity:

```
accessor.c:44:5: warning: macro expansion producing 'defined' has undefined behavior [-Wexpansion-to-defined]
#if DESIGNATED_INITIALIZERS
    ^
```
and
```
accessor.c:108:5: warning: macro expansion producing 'defined' has undefined behavior [-Wexpansion-to-defined]
#if DESIGNATED_INITIALIZERS
    ^
```